assemble-appauth.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+cd $1
+
+if [ ! -f $2libs/appauth-release.aar ]; then
+    ./gradlew assembleRelease
+    mkdir $2/libs
+    cp $1/library/build/outputs/aar/appauth-release.aar $2/libs/appauth-release.aar
```

Comment:
Does this mean downstream consumers of this library will need to grab this `aar` too?

Reply:
Nope they don't need to change anything on their side